download_and_install/ola_on_raspberry_pi.md

```diff
@@ -0,0 +1,68 @@
+OLA on Raspberry Pi
+===================
+
+This tutorial describes how to get OLA running on the
+[Raspberry Pi](https://www.raspberrypi.org/). The procedure described here is designed to get OLA up and running as fast
```

Comment:
or instead "as fast as possible": "the easiest way possible". It's not about speed, it's about less work ;)

Reply:
It will also be faster than compiling!